lib/features/ets/quick-link/quick_links_view.dart

```diff
@@ -71,7 +71,7 @@ class _QuickLinksViewState extends State<QuickLinksView>
             ),
             Expanded(
               child: _buildReorderableGridView(
-                  model, model.deletedQuickLinks, _buildAddButton),
+                  model, model.deletedQuickLinks, _buildAddButton, true),
```

Comment:
Pourrais-tu ajouter le nom du paramètre pour éviter d'avoir un booléen magique ?
```suggestion
                  model, model.deletedQuickLinks, _buildAddButton, blockReorder: true),
```